RawVideo/RawVideo/ContentView.swift

```diff
@@ -0,0 +1,1032 @@
+                                .background(Color(.red))
+                                .cornerRadius(5)
+                            }
+                        }
+                    }
+                }
+                .frame(width: geometryReader.size.width, height: geometryReader.size.height)
+                .onAppear(perform: {
+                    let defaults = UserDefaults.standard
+                    let savedToken = defaults.value(forKey: "Token")
+                    
+                    if (savedToken != nil)
+                    {
+                        token = savedToken as! String
+                    }
+                })
+                .onTapGesture {
+                    if (isFocused)
+                    {
+                        isFocused = false
+                    }
+                }
+                .disabled(loading)
+                .zIndex(0)
+            }
+        }
+    }
+
+    struct ContentView_Previews: PreviewProvider 
+    {
+        static var previews: some View
+        {
+            ContentView()
+        }
+    }
+    
+    private func InitResources() async -> Void
+    {
+        localVideoStreamObserver = LocalVideoStreamObserver(view: self)
+        virtualRawOutgoingVideoStreamObserver = VirtualRawOutgoingVideoStreamObserver(view: self)
+        screenShareRawOutgoingVideoStreamObserver = ScreenShareRawOutgoingVideoStreamObserver(view: self)
+        remoteVideoStreamObserver = RemoteVideoStreamObserver(view: self)
+        rawIncomingVideoStreamObserver = RawIncomingVideoStreamObserver(view: self)
+        rawIncomingVideoStreamObserver?.delegate = OnRawVideoFrameArrived
+        remoteParticipantObserver = RemoteParticipantObserver(view: self)
+        callObserver = CallObserver(view: self, remoteParticipantObserver: remoteParticipantObserver!)
+        
+        await CreateAgent()
+        
+        if (deviceManager == nil)
+        {
+            return
+        }
+        
+        videoDeviceInfoList = deviceManager?.cameras ?? []
+        videoDeviceInfoItemList = [VideoDeviceInfoItem]();
+        selectedVideoDeviceInfoIndex = videoDeviceInfoList.count > 0 ? 0 : -1;
+        
+        for i in 0 ..< videoDeviceInfoList.count
+        {
+            let videoDeviceInfoItem = VideoDeviceInfoItem(id: i, name: videoDeviceInfoList[i].name)
+            videoDeviceInfoItemList.append(videoDeviceInfoItem)
+        }
+        
+        cameraList = CameraCaptureService.GetCameraList()
+        cameraItemList = [CameraItem]();
+        selectedCameraIndex = cameraList.count > 0 ? 0 : -1;
+        
+        for i in 0 ..< cameraList.count
+        {
+            let cameraItem = CameraItem(id: i, name: cameraList[i].localizedName)
+            cameraItemList.append(cameraItem)
+        }
+        
+        showCallSettings = true
+        
+        let defaults = UserDefaults.standard
+        let savedMeetingLink = defaults.value(forKey: "MeetingLink")
```

Comment:
Can use `string(forKey:` so no need to cast with `as! String` later
```suggestion
        let savedMeetingLink = defaults.string(forKey: "MeetingLink")
```